test/gtest/ucp/test_ucp_request.cc

```diff
@@ -377,17 +377,23 @@ class test_proto_reset : public ucp_test {
                         unsigned reqs_count = 1000,
                         size_t msg_size = UCS_KBYTE * 70)
     {
-        for (int i = 0; i < reqs_count; ++i) {
+        unsigned warmup_req_count = 5;
+
+        for (unsigned i = 0; i < ucs_max(reqs_count, warmup_req_count); ++i) {
             mapped_buffer *rbuf = new mapped_buffer(msg_size, receiver());
             rbuf->memset(0);
             m_rbufs.push_back(std::unique_ptr<mapped_buffer>(rbuf));
             m_sbufs.push_back(std::vector<uint8_t>(msg_size));
         }
 
-        /* Send a single message to complete wireup before sending actual
+        /* Send a bunch of messages to complete wireup before sending actual
            data */
+        /* TODO: a single message is not enough to complete wireup with shm,gga
+         *       TLs set, should not we be able to restart request with wireup
+         *       message as well?
```

Comment:
why one is not enough? maybe need to flush or wait for receive?

Reply:
AFAIR flush did not help, should not sync transfer guarantee request completion on receiver side as well? 
I don't see much difference with 1 or 5 warmup requests for this specific test, how about restart of wireup msg request? Cannot it be a real problem?

Reply:
it's not a problem to have several wireup messages, but the comment here is not clear, and why 5 is enough and not 50? and why the problem is uncovered after adding gga, is it slower to ininitialize?